services/web/server/src/simcore_service_webserver/users/_preferences_models.py

```diff
@@ -88,6 +88,13 @@ class JobConcurrencyLimitFrontendUserPreference(FrontendUserPreference):
     value: int | None = 1
 
 
+class TelemetryLowDiskSpaceWarningThresholdFrontendUserPreference(
+    FrontendUserPreference
+):
+    preference_identifier = "lowDiskSpaceThreshold"
```

Comment:
No annotation in preference_identifier? i

Reply:
It overwrites an existing field, which has the annotation already defined.
I think it's not required, since it was already defined.

If provided or not, the tooling does not complain. I have no strong opinion on this.

Reply:
I would in the case of BaseModels. These are **not** class attributes in the classic sense!